examples/connectivity/plot_mne_inverse_label_connectivity.py

```diff
@@ -49,6 +56,27 @@
                     baseline=(None, 0), reject=dict(mag=4e-12, grad=4000e-13,
                                                     eog=150e-6))
 
+###############################################################################
+# Compute inverse solutions and their connectivity
+# ------------------------------------------------
+#
+# Next, we need to compute the inverse solution for this data. This will return
+# the sources / source activity that we'll use in computing connectivity.
+#
+# Now we are ready to compute the connectivity in the alpha band. Notice
```

Comment:
maybe connect this sentence with the one above for example:
"that we'll use in computing connectivity in the alpha band."